src/main/java/com/github/tomakehurst/wiremock/core/Options.java

```diff
@@ -136,4 +136,6 @@ default Function<Extensions, NotMatchedRenderer> getNotMatchedRendererFactory()
   Set<String> getTemplatePermittedSystemKeys();
 
   boolean getTemplateEscapingDisabled();
+
+  boolean failIfMultipleMappingsMatch();
```

Comment:
Needs a default return if we do not want to break binary compatibility of a public interface

Reply:
Also should be `failIfMultipleMappingsMatch()` to follow the pattern

Reply:
I think this is fixed